### ISSUE TITLE
Unable to parse services.yaml for the emporia_vue integration

### ISSUE THREAD

**Issue by jamieFL** [magico13/ha-emporia-vue#207]
New error in my logs.  My entities are visible and showing values.  
```
Logger: homeassistant.helpers.service
Source: helpers/service.py:554
First occurred: 18:39:07 (1 occurrences)
Last logged: 18:39:07

Unable to parse services.yaml for the emporia_vue integration
```

**Comment by shayes83** [comment]
Also showed up for me today

**Comment by magico13** [comment]
Resolved with version 0.8.2
